content/20.treatments.md

```diff
@@ -302,8 +302,17 @@ All of these factors could potentially influence treatment outcome.
 Furthermore, the authors acknowledge that the effect of the drugs might be different in females and pediatric subjects, since these subjects were not the part of the study.
 
 **Summary.**
-In vitro evidence shows HCQ may be an effective therapeutic against SARS-CoV-2 and COVID-19.
-Multiple clinical studies have already been carried out to assess this possibility.
+_In vitro_ evidence suggests that HCQ may be an effective therapeutic against SARS-CoV-2 and COVID-19. 
+As recently discussed [@doi:10.1093/jac/dkaa114], because the 90% effective concentration (EC~90~) of CQ in Vero E6 cells (6.90 μM) can be achieved in and tolerated by rheumatoid arthritis patients, it may also be possible to achieve the effective concentration in COVID-19 patients.
+Additionally, HCQ was found to be effective in HIV [@doi:10/cq2hx9] and chronic Hepatitis C [@doi:10.1002/jmv.24575] patients.
+Therefore, initially it was thought that it would be effective towards SARS-COV-2.
+The hypotheses were that HCQ could alleviate the development of COVID-19 through mitigation of CD154 in T cells,  could have the same effect as an antiviral like CQ pre- and post-infection, is cheaper, and has less side effects than CQ. 
```

Comment:
```suggestion
```
@soumitagh is this what you meant? I definitely don't think you need to cut this section unless you want to! I think it's useful info for the summary